data_pipelines_cli/cli_commands/deploy.py

```diff
@@ -196,6 +201,12 @@ def _sync_bucket(self) -> None:
     required=False,
     help="Authorization OIDC ID token for a service account to communication with cloud services",
 )
+@click.option(
+    "--disable-bucket-sync",
+    is_flag=True,
+    default=True,
```

Comment:
Shouldn't `false` be the default value? In the description of this PR you wrote that now we have a flag to disable the bucket sync. I think there is some inconsistency here: in one place we have disable-bucket-sync but it's value is passed to the sync-bucket variable directly? idk

Reply:
The default value of "false" will result in a feature that is not backwards compatible. This means that all users will need to update their cicd workflows to accommodate this change. As a solution, @p-pekala has suggested creating a flag that can disable the sync bucket. This will offer the possibility of disabling the feature while still maintaining backwards compatibility.